docs/blitz/design/actions/actions_list.rst

```diff
@@ -404,6 +404,53 @@ snapshot you can specify to not delete it. See `example` below.
         device: my_device
         delete_snapshot: False #optional, default is True
 
+yang_snapshot
+^^^^^^^^^^^^^
+
+The ``yang_snapshot`` action is another alternative to capture related device
+configuration in a lightweight way. It does not save all device configuration,
+instead, it scans Genie Trigger information and find out ``yang`` actions that
+will change the device configuration before the next ``yang_snapshot_restore``
+action. Note that one ``yang_snapshot`` action can be followed by one or
+multiple ``yang_snapshot_restore`` actions. When the related ``yang`` actions
+are known, ``yang_snapshot`` action is able to snapshot partial device
+configuration, which will be restored when the ``restore_config_snapshot``
+action comes.
```

Comment:
When you begin to describe multiple ``yang_snapshot_restore`` usage I get confused.  Why is ``restore_config_snapshot`` related?

Reply:
Say there is a `yang` action of `/ios:native/ios:ntp` before the first `yang_snapshot_restore`, and then there is another `yang` action of `/oc-if:interfaces` before the second 'yang_snapshot_restore'. The action `yang_snapshot` should collect a snapshot of `/ios:native` and `/oc-if:interfaces`. When I perform `yang_snapshot`, all following `restore_config_snapshot` before the next `yang_snapshot` are considered.